docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,53 @@
+Canonical URLs
+==============
+
+Canonical URLs allow you to have consistent page URLs across all your domains.
+This is mainly useful for search engines,
+so that they can send users to the correct version and domain of your documentation.
+
+The canonical URL is generated based on:
+
+* The default version of your project (usually "latest" or "stable").
+* The canonical :doc:`custom domain </custom-domains>` if you have one,
+  otherwise the default :ref:`subdomain <hosting:subdomain support>` will be used.
+
+.. contents::
+    :local:
+
+Example
+-------
+
+Fabric hosts their docs on Read the Docs,
+but they use their own domain ``http://docs.fabfile.org``.
+This means that Google will index both ``http://fabric-docs.readthedocs.io`` and
+``http://docs.fabfile.org`` for their documentation.
+
+Fabric will want to set ``http://docs.fabfile.org`` as their canonical domain,
+this means that when Google indexes ``http://fabric-docs.readthedocs.io``,
+it will know that it should really point at ``http://docs.fabfile.org``.
```

Comment:
I think it will be useful to merge this example section with the previous bullets and generalize the example without using Fabric but just `example.com` or similar.